embark/reporter/tests.py

```diff
@@ -0,0 +1,127 @@
+import secrets
+from http import HTTPStatus
+from django.test import TestCase
+from django.test import Client
+
+from users.models import User
+from uploader.models import FirmwareAnalysis, LogZipFile
+
+
+class TestAPI(TestCase):
+    def __init__(self, methodName='runTest'):
+        super().__init__(methodName)
+        self.client = Client()
+        self.user = None
+        self.superuser = None
+
+        self.analysis1 = None
+        self.analysis2 = None
+        self.analysis3 = None
+        self.analysis4 = None
+
+    def setUp(self):
+        self.client = Client()
```

Comment:
Specify the API key here. Otherwise, you have to pass the API on every request manually
```suggestion
        self.client = Client(headers={"Authorization": self.user.api_key})
```